app/user.go

```diff
@@ -505,11 +505,11 @@ func (a *App) GetUsersInChannel(channelId string, offset int, limit int) ([]*mod
 }
 
 func (a *App) GetUsersInChannelByStatus(channelId string, offset int, limit int) ([]*model.User, *model.AppError) {
-	result := <-a.Srv.Store.User().GetProfilesInChannelByStatus(channelId, offset, limit)
-	if result.Err != nil {
-		return nil, result.Err
+	result, err := a.Srv.Store.User().GetProfilesInChannelByStatus(channelId, offset, limit)
```

Comment:
You can simplify it using `return a.Srv.Store.User().GetProfilesInChannelByStatus(channelId, offset, limit)` directly.